src/charm.py

```diff
@@ -345,11 +345,15 @@ def configure_profile(self, profile_name):
         except RetryError:
             self.log.error(f"Action failed. namespace {profile_name} was not created")
             return
-
-        for file in PROFILE_CONFIG_FILES:
-            # TODO figure out which integrations are needed
-            yaml_text = self._safe_load_file_to_text(file)
-            self._apply_manifest(yaml_text, profile_name)
+        try:
+            for file in PROFILE_CONFIG_FILES:
+                # TODO figure out which integrations are needed
+                yaml_text = self._safe_load_file_to_text(file)
+                self._apply_manifest(yaml_text, profile_name)
+        except LoadResourceError as e:
+            self.log.error(
```

Comment:
should not we raise an error in this scenario?

Reply:
I don't think the unit should go into error due to an action not being handled, what do we think?

Reply:
agreed